mesonbuild/modules/cuda.py

```diff
@@ -266,6 +311,22 @@ def _nvcc_arch_flags(self, cuda_version: str, cuda_arch_list: AutoArch, detected
             if version_compare(cuda_version, '<13'):
                 cuda_hi_limit_gpu_architecture = '10.0'       # noqa: E221
 
+        if version_compare(cuda_version, '>=12.9'):
+            cuda_known_gpu_architectures.append('Blackwell')
+            cuda_common_gpu_architectures.extend(['10.0', '10.3', '12.0', '12.1'])
+            cuda_all_gpu_architectures.extend(['10.0', '10.3', '12.0', '12.1'])
```

Comment:
Missing `10.1` in both these?

https://docs.nvidia.com/cuda/archive/12.9.0/cuda-compiler-driver-nvcc/#virtual-architecture-feature-list